project/src/components/app/app.tsx

```diff
@@ -1,5 +1,12 @@
-function App(): JSX.Element {
-  return <p>Hello, world!</p>;
+import WelcomeScreen from '../../pages/welcome-screen/welcome-screen';
+import {WelcomePageProps} from '../../types/props/welcome-page';
+
+function App(prop: WelcomePageProps): JSX.Element {
```

Comment:
Обычно используют:
1. function expression для описания компонента
2. `prop` → `props` так как их все таки "много"

```suggestion
const App = (props: WelcomePageProps): JSX.Element  => {
```

Reply:
done